de.tu_bs.cs.isf.e4cf.parts.project_explorer/src/de/tu_bs/cs/isf/e4cf/parts/project_explorer/ProjectExplorerLabelProvider.java

```diff
@@ -52,34 +55,36 @@ public boolean isLabelProperty(Object element, String property) {
 
 	@Override
 	public void removeListener(ILabelProviderListener listener) {
-		 if (listeners.contains(listener)) { 
-			  listeners.remove(listener); 
-		 } 
+		if (listeners.contains(listener)) {
+			listeners.remove(listener);
+		}
 	}
+
 	/**
 	 * This method gives the different files a different look
 	 */
 	@Override
 	public Image getImage(Object element) {
 		Image image = null;
-		
+
 		if (element instanceof FileTreeElement) {
 			FileTreeElement fileElement = (FileTreeElement) element;
-			if(workspaceFileSystem.isProject(fileElement)) {
+			if (workspaceFileSystem.isProject(fileElement)) {
 				image = imageService.getImage(null, "icons/Explorer_View/items/project16.png");
 			} else if (fileElement.isDirectory()) {
 				image = imageService.getImage(null, "icons/Explorer_View/items/folder16.png");
 			} else {
 				String fileExtension = fileElement.getExtension();
-				//load extended file icons 
-				if(fileExtensions.containsKey(fileExtension)) {
+				// load extended file icons
+				if (fileExtensions.containsKey(fileExtension)) {
 					image = fileExtensions.get(fileExtension).getIcon(imageService);
 				} else if (fileExtension.equals(E4CStringTable.FILE_ENDING_XML)) {
 					image = imageService.getImage(null, "icons/Explorer_View/items/xml16.png");
 				} else {
-					//default file icon
+					// default file icon
 					image = imageService.getImage(null, "icons/Explorer_View/items/file16.png");
-				};
+				}
+				;
```

Comment:
Are you sure to let that semicolon there?

Reply:
Resolved :)